src/OpenDebugAD7/TextPositionTuple.cs

```diff
@@ -24,30 +36,132 @@ private TextPositionTuple(Source source, int line, int column)
         public static TextPositionTuple GetTextPositionOfFrame(PathConverter converter, IDebugStackFrame2 frame)
```

Comment:
>GetTextPositionOfFrame [](start = 40, length = 22)

Similar to my suggestion in the other review - you don't want to do this here, but rather in HandleStackTraceRequestAsync because there is no need to do this stuff for stopping event processing.

Reply:
Well here it's deliberate, sort of. VSCode didn't use the stopped event source but maybe other clients do?

Reply:
The source in the stopped event was actually an extension added long ago to support testing. It should be fine to just use the original path.

Reply:
Ha, something to add to the spec.